Contracts/Grains/Storage/IStorageGrain.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Newtonsoft.Json.Linq;
+using Orleans;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace CommunAxiom.Commons.Client.Contracts.Grains.Storage
+{
+    public interface IStorageGrain : IGrainWithStringKey
+    {
+        Task SaveData(JObject obj);
+        Task<List<JObject>> GetData();
```

Comment:
As in the grain, this should return a single JObject because that's what we're saving in the grain